bin/build-sprite-string.js

```diff
@@ -0,0 +1,32 @@
+import defaultAttrs from '../src/default-attrs.json';
+
+const svgStartTag = `<svg xmlns="${defaultAttrs.xmlns}">
+  <defs>\n`;
+const svgEndTag = ' </defs>\n</svg>';
+
+/**
+ * Renders the inner sprites as SVG Symbols
+ * @param {object} icons the icons object
+ * @returns {string} the rendered string with SVG symbols
+ */
+function buildSpriteString(icons) {
+  const symbols = Object.keys(icons)
+    .map(icon => toSvgSymbol(icon, icons[icon]))
+    .reduce((spriteString, symbolString) => spriteString + symbolString, '');
```

Comment:
Could we just use `.join('')` here instead?

Reply:
Sure.